app/harvesters/Itms/accounting_documents_harvester.rb

```diff
@@ -0,0 +1,92 @@
+require 'harvester_utils/downloader'
+
+class Itms::AccountingDocumentsHarvester
```

Comment:
Nevidím žádné výhody v tom, že se stejný kód ve všech třídách opakuje a to ani v případě budoucích změn. Možná to tak můžeš implementovat nyní rychleji, ale člověk který bude projekt udržovat a dále rozšiřovat by tě za takové rozhodnutí asi nepochválil.